Sources/ShowAttachments.php

```diff
@@ -110,6 +110,7 @@ function showAttachment()
 		$smcFunc['db_free_result']($request);
 
 		$file['filePath'] = getAttachmentFilename($file['filename'], $attachId, $file['id_folder'], false, $file['file_hash']);
+		$file['filePath'] = !file_exists($file['filePath']) ? substr($file['filePath'], 0, -(strlen(pathinfo($file['filePath'])['extension'])+1)) : $file['filePath'];
```

Comment:
I forgot about that change to PHP.
I'm glad they changed it because 2 lines of code to do one task seems superfluous.